hashes/src/siphash24.rs

```diff
@@ -14,6 +14,21 @@ use crate::HashEngine as _;
 #[repr(transparent)]
 pub struct Hash([u8; 8]);
 
+#[cfg(feature = "schemars")]
+impl schemars::JsonSchema for Hash {
+    fn schema_name() -> String { "Hash".to_owned() }
+
+    fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> schemars::schema::Schema {
+        let mut schema: schemars::schema::SchemaObject = <String>::json_schema(gen).into();
+        schema.string = Some(Box::new(schemars::schema::StringValidation {
+            max_length: Some(8 * 2),
+            min_length: Some(8 * 2),
+            pattern: Some("[0-9a-fA-F]+".to_owned()),
+        }));
+        schema.into()
+    }
+}
```

Comment:
I'd rather fix it by splitting the macro since that will just blanket-solve everything related to that problem.